### PR TITLE
Add ROI harness for the X1Y0 clock region

### PR DESCRIPTION
**Pull request by HackerFoo**
This adds a simple harness in the X1Y0 clock region (bottom right) where two inputs and two outputs are connected to the closest corresponding LUT.

This is helpful to understand IO.

- tested with roi_inv.v on hardware

Screenshot from 2019-07-24 17-17-58

### CODE REVIEW

File: minitests/roi_harness/basys3-x1y0-fixed.xdc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+set_property LOC SLICE_X64Y39 [get_cells {roi/ins[1].lut}]
+
+## outs
+set_property BEL A6LUT [get_cells {roi/outs[0].lut}]
+set_property LOC SLICE_X64Y38 [get_cells {roi/outs[0].lut}]
+set_property BEL A6LUT [get_cells {roi/outs[1].lut}]
+set_property LOC SLICE_X64Y37 [get_cells {roi/outs[1].lut}]
+
+# fix routes
+set_property FIXED_ROUTE { { IOB_IBUF0 RIOI_I0 RIOI_ILOGIC0_D IOI_ILOGIC0_O IOI_LOGIC_OUTS18_1 INT_INTERFACE_LOGIC_OUTS18 WL1BEG_N3 NW2BEG0 IMUX7 CLBLM_M_A1 }  } [get_nets {din_IBUF[0]}]
```
Comment (litghost) [inline minitests/roi_harness/basys3-x1y0-fixed.xdc]: Why is this required at all?  No IO is required from the ROI at all?

**Comment by litghost** [comment]
> This is helpful to understand IO.

So why make an ROI harness at all?  Why not just make a minitests?

File: minitests/roi_harness/basys3-x1y0-fixed.xdc (hunk 1 above)
Comment (HackerFoo) [reply minitests/roi_harness/basys3-x1y0-fixed.xdc]: So that I can test that I haven't broken anything by moving the ROI.
